### DISCUSSION TITLE
Blueprinter support

### DISCUSSION THREAD

**Discussion by HotFusionMan** [ElMassimo/types_from_serializers#10] [Ideas]
I'm curious how feasible it would be to add support for Blueprinter based serializers. Their API differs enough from what's already supported that it seems like it could be a lot of effort.

**Comment by PedroAugustoRamalhoDuarte** [comment]
Hello @HotFusionMan, we are talking about that is this discussion https://github.com/ElMassimo/types_from_serializers/discussions/11
